### ISSUE TITLE
Not possible to list price for half or less "x" currency?

### ISSUE THREAD

**Issue by benkei36** [testpushpleaseignore/acquisition#20]
It seems everything that i try to sell for 1/2 a chaos or 3/2 etc, defaults to 0 chaos pricing. Am i missing something? Or is some sort of limitation that can't be fixed

**Comment by benkei36** [comment]
Yes i noticed and tried that also, but if i put it for like <0,1 it won't show up in the trade site at all

**Comment by testpushpleaseignore** [comment]
Oh interesting, yeah it seems that decimal prices below a certain point don't actually get a price in the POE Trade. I can look to see if it's possible fractionalize the set price in Acquisition when uploading it to the forums. Not sure how long it might take.

**Comment by benkei36** [comment]
No biggie, that'd be great
